docker/apache/openoni.conf

```diff
@@ -7,29 +10,12 @@ WSGIProcessGroup openoni-wsgi-app
 #ServerAdmin you@example.com
 #ServerName localhost
 
-WSGIScriptAlias / /opt/openoni/conf/openoni.wsgi
-
 RewriteEngine  on
 
 RewriteRule ^/share/(.*) http://loc.gov/share/$1 [P] 
 
 RewriteRule ^/data/(.+)/(.+)sample(.+) - [F]
 
-RewriteRule ^/index.html / [redirect=permanent,last]
```

Comment:
I'm pretty sure nothing is using *.html anymore.  The other paths may be in use in a hard-coded way, but I *really* hope not.  I'd honestly rather break it and see what we need to fix than keep it "just in case".  But I'll see if I can find any references.

To be fair, this totally doesn't belong here.  It was confusing me when trying to search for what things needed to be hacked, unhacked, etc. to get the new structure working.